index.html

```diff
@@ -6,6 +6,6 @@
   <title>Document</title>
 </head>
 <body>

```

Comment:
```suggestion
<body>
<body class="body">
```

En todas las páginas siempre ponemos esta clase al body